src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs

```diff
@@ -209,6 +248,192 @@ private UserTypeTypeParametersFormatDefinition()
         }
         #endregion
 
+        #region User Symbols - Field Name 
+        [Export(typeof(EditorFormatDefinition))]
+        [ClassificationType(ClassificationTypeNames = ClassificationTypeNames.FieldName)]
+        [Name(ClassificationTypeNames.FieldName)]
```

Comment:
❓ Why 'Field Name' instead of 'Fields'? The latter appears to be a better match for existing items like 'Classes' and 'Delegates'.